MCprep_addon/mcprep_ui.py

```diff
@@ -1044,6 +1053,12 @@ def draw(self, context):
 					# datapass = scn_props.mob_list[mob_ind].mcmob_type
 					tx = f"Spawn {name} with {skinname}"
 					row.operator("mcprep.spawn_with_skin", text=tx)
+			b_row.label(text="Resource pack")
```

Comment:
Same story about i18n
```suggestion
			b_row.label(text=env._("Resource pack"))
```

Reply:
Yeah, the code was originally written months ago. I hold this for awhile and stuck there so. So noted.